modules/oadp-installing-dpa.adoc

```diff
@@ -57,6 +57,8 @@ spec:
         - aws
     restic:
       enable: true <.>
+      PodConfig
```

Comment:
this should be `podConfig`

Reply:
@dymurray 
It's `PodConfig` here: https://github.com/openshift/oadp-operator/blob/master/docs/API_ref.md#resticconfig
Error?